.github/workflows/build.yml

```diff
@@ -57,7 +57,13 @@ jobs:
           --health-retries 5
     strategy:
       matrix:
-        python-version: [ 3.6, 3.7, 3.8, 3.9 ]
+        django-version: [3.2, 4.0]
+        python-version: [ 3.8, 3.9, '3.10' ]
```

Comment:
[Style] The array style should be consistent `[ 3.8, 3.9, '3.10' ]` or `[3.8, 3.9, '3.10']`.
Pick your style

Reply:
[Style] Are the single quotes necessary around `'3.10'`

Reply:
I guess the way to go is :
`django-version: [ 3.2, 4.0 ]`
to be consistent with the rest of the file

Reply:
> [Style] Are the single quotes necessary around `'3.10'`

Python 3.10 is not recognized if we don't use the quotes. The interpreter used python 3.1 instead of python 3.10